bin/OSPData/adera/ph_rcs.sturdy.gltf

```diff
@@ -57,6 +57,9 @@
                 "description" : "A placeholder linear RCS block",
                 "manufacturer" : "ACME Placeholders",
                 "machines" : [
+                    {
```

Comment:
Completely unrelated to your work here, but it would amuse me enormously if someone were to create a pure constexpr gltf file parser. 

There's a pure constexpr json parser out there, so it should be possible (or nearly so anyway)

Reply:
Oh man... how would one go about doing this? Sounds like an interesting exercise

Reply:
Start here: https://www.youtube.com/watch?v=PJwd4JLYJJY

it would probably be useful for commercial projects that absolutely must have initial application startup as fast as fucking possible